deploy/001_deploy_hubpool.ts

```diff
@@ -24,4 +28,4 @@ const func = async function (hre: any) {
   });
 };
 module.exports = func;
-func.tags = ["hubpool"];
+func.tags = ["HubPool", "mainnet"];
```

Comment:
Updated the tags so they're all the same style and so that it's easy to deploy all the contracts needed for a particular network.